Functions/Verbs/msvc90/script.js

```diff
@@ -0,0 +1,25 @@
+include(["Functions", "Engines", "Wine"]);
+include(["Functions", "Net", "Resource"]);
+include(["Functions", "Filesystem", "Extract"]);
+include(["Functions", "Verbs", "luna"]);
+
+Wine.prototype.msvc90 = function() {
```

Comment:
I don't get it. Right now it seemed to work with vcrun2008. Did however not reinstall. So maybe something got mixed up.